apps/web/src/features/finance/actions/getPayments.ts

```
@@ -0,0 +1,643 @@
+    console.error("Unexpected error in getPayment:", error);
+    return null;
+  }
+}
+
+export async function getAllPayments() {
+  try {
+    const supabase = await createClient();
+
+    const { data, error } = await supabase
+      .from("payments_with_details" as any) // Using 'as any' temporarily until view is created
+      .select(`*`)
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)